snapshot/src/test/java/io/zeebe/snapshots/impl/SnapshotChecksumTest.java

```diff
@@ -128,4 +141,24 @@ public void shouldDetectCorruptedSnapshot() throws IOException {
     // then
     assertThat(SnapshotChecksum.verify(corruptedSnapshot)).isFalse();
   }
+
+  @Test
+  public void shouldCalculateSameChecksumOfLargeFile() throws IOException {
+    // given
+    final var largeSnapshot = temporaryFolder.newFolder().toPath();
+    final Path file = largeSnapshot.resolve("file");
+    final String largeData = "a".repeat(1000 * 1000 * 4);
```

Comment:
it's probably fine if it's at least a few times the block size, and it's not a multiple of it, no?